core/src/main/java/de/danoeh/antennapod/core/storage/APCleanupAlgorithm.java

```diff
@@ -88,7 +88,7 @@ Date calcMostRecentDateForDeletion(@NonNull Date currentDate) {
     @NonNull
     private List<FeedItem> getCandidates() {
         List<FeedItem> candidates = new ArrayList<>();
-        List<FeedItem> downloadedItems = DBReader.getDownloadedItems();
+        List<FeedItem> downloadedItems = DBReader.getDownloadedItems(null);
```

Comment:
I would specify the default here instead of passing null. Makes it a bit more explicit from within the CleanupAlgorithm about what it internally expects.